packages/expo-maps/src/apple/AppleMapsView.tsx

```diff
@@ -31,6 +31,9 @@ export const AppleMapsView = React.forwardRef<AppleMapsViewType, AppleMapsViewPr
       setCameraPosition(config?: CameraPosition) {
         nativeRef.current?.setCameraPosition(config);
       },
+      async openLookAroundAsync(coordinates: Coordinates) {
+        await nativeRef.current?.openLookAroundAsync(coordinates);
```

Comment:
```suggestion
       nativeRef.current?.openLookAroundAsync(coordinates);
```
The await is redundant here